libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI+Blocks.h

```diff
@@ -131,6 +131,26 @@ typedef void (^SFRestBatchResponseBlock) (SFSDKBatchResponse *response, NSURLRes
                                         failBlock:(SFRestFailBlock)failBlock
                                     completeBlock:(SFRestDictionaryResponseBlock)completeBlock NS_SWIFT_UNAVAILABLE("Use RestRequest factory methods to construct the request and RestClient send instead.");
 
+/**
+ * Returns an `SFRestRequest` object that provides layout data for the specified parameters.
+ * @param objectAPIName Object API name.
+ * @param formFactor Form factor. Could be "Large", "Medium" or "Small". Default value is "Large".
+ * @param layoutType Layout type. Could be "Compact" or "Full". Default value is "Full".
+ * @param mode Mode. Could be "Create", "Edit" or "View". Default value is "View".
+ * @param recordTypeId Record type ID. Default will be used if not supplied.
+ * @param failBlock Failure block.
+ * @param completeBlock Completion block.
+ * @see https://developer.salesforce.com/docs/atlas.en-us.uiapi.meta/uiapi/ui_api_resources_record_layout.htm
+ */
+- (SFRestRequest *) performLayoutWithObjectAPIName:(NSString *)objectAPIName
```

Comment:
Just like the comment above, would a request object  with a builder be better suited for these apis? We have done similar changes with Composite and Batch.

Reply:
Sounds good - maybe a future enhancement. For now, I'll leave this as-is to stay consistent with the `Android` API.